config/webpack/development.js

```diff
@@ -1,5 +1,16 @@
-process.env.NODE_ENV = process.env.NODE_ENV || 'development'
+const dotenv = require('dotenv');
 
-const environment = require('./environment')
+const paths = [
+  '.env.local',
+  `.env.${process.env.NODE_ENV}`,
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'process' is not defined  no-undef